pkg/utils/common.go

```diff
@@ -141,6 +150,13 @@ var (
 		Version:  corev1.SchemeGroupVersion.Version,
 		Resource: "services",
 	}
+
+	// TO-DO (chenyu1): Add more common GVRs/GVKs here.
+	CRPV1Beta1GVK = schema.GroupVersionKind{
```

Comment:
```suggestion
	CRPGVK = schema.GroupVersionKind{
```
to avoid the variable rename in the future when we upgrade the API version.

Reply:
It's called a GVK so different versions = different objects. To avoid breaking it might be easier to add a new one rather than reusing the same.